### PR TITLE
#25169 Apps Framework README > Split out admin content to child pages

### PR DESCRIPTION
**Pull request by Reene-Simon**
#### Summary

This PR organizes the README section as per the instructions in the issue description. I have used the reviews given for a similar issue, namely the Bitbucket README and have tried to incorporate the instructions given by the reviewers. These are the main changes that have been done:

1. Created a table of contents as mentioned
2. Made a new admin-guide file under the docs folder
3. Moved all configuration details into the admin-guide file
4. Linked all the links and the table of contents according to the changes made

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/25169

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thanks, @Reene-Simon!

File: docs/admin-guide.md
Comment (mickmister) [inline docs/admin-guide.md:33]: @cwarnermm There isn't really any "admin" documentation here. This is all documentation on how to run the unit and e2e tests for the project. Though I'm thinking there are other improvements we can make here:

- Keep everything in README.md since there isn't much to extract related to admin here
- Move "Deploying and Installing Apps" further up in the README, before the "Getting Started" section
- Rename "Getting Started" to "Develop an App"
- Remove the "Contacts" section at the bottom of the README

**Review by mickmister — COMMENTED** [review]
Thanks for the contribution @Reene-Simon!

Please let me know your thoughts on my comment below pertaining to moving the content around a bit

File: docs/admin-guide.md
Comment (Reene-Simon) [reply docs/admin-guide.md:33]: I was thinking how about adding the contacts above the table of contents? Like how it is in the other README's ? Like maintainer and co-maintainer and their GitHub profile links? If they are actually the maintainers? Sorry I'm new here so I don't know much about this.

Comment (mickmister) [reply docs/admin-guide.md:33]: > If they are actually the maintainers?

No unfortunately we don't have a specific list of maintainers to put into the README today

Comment (Reene-Simon) [reply docs/admin-guide.md:33]: Ok sure. Will make the changes now

**Comment by Reene-Simon** [comment]
@cwarnermm @mickmister I have made the requested changes. Please do review it.

File: README.md
Comment (mickmister) [inline README.md:50]: This section and "Develop an App" should go above the "Running the Tests" section

Comment (Reene-Simon) [reply README.md:50]: Ok sure

Comment (Reene-Simon) [reply README.md:50]: @mickmister should I remove the end user title?
Because I have added this section below the same

Comment (Reene-Simon) [reply README.md:50]: @mickmister I'll commit it for now and do let me know if u want this change.

Comment (mickmister) [reply README.md]: Sorry, to be more clear - "Deploying and Installing Apps" should also go above "Develop an App". And yes let's remove the "end user" title

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
Approving assuming the above change is done

**Comment by Reene-Simon** [comment]
@cwarnermm @mickmister I have made the relevant changes. Do check it out

**Pull request merged**
